setup.py

```diff
@@ -27,7 +27,7 @@
     [
         'numpy',
         'h5py',
-        'ruamel.yaml',
+        'ruamel.yaml<15',
```

Comment:
Just double checking that this should be 15 and not .15

Reply:
Good catch.